src/helm-util.ts

```diff
@@ -108,12 +108,11 @@ export function findHelm(rootFolder: string): string {
 
 export async function getHelmPath() {
     var helmPath = "";
-    if (core.getInput('helm-version', { required : false })) {
-        var version = core.getInput('helm-version', { required: false });
+    var version = core.getInput('helm-version', { required: false });
```

Comment:
use const here. if you can't use const when declaring a variable you should pretty much always use let.